UI/src/view/navigationpanel/ActionPanelBoxes.js

```diff
@@ -161,6 +161,103 @@
     };
   };
 
+  ActionPanelBoxes.WinterSpeedLimitBox = function(asset) {
+    var speedLimits = [100, 80, 70, 60];
+    var speedLimitLegendTemplate = _.map(speedLimits, function(speedLimit) {
+      return '<div class="legend-entry">' +
+        '<div class="label">' + speedLimit + '</div>' +
+        '<div class="symbol linear speed-limit-' + speedLimit + '" />' +
+        '</div>';
+    }).join('');
```

Comment:
The used css style is the same, but I can create a new style for winter speed limits and change this. 
This should be commented on code review of : ` Droth 705 labeling of linear assets (winter speed limits)`